CMakeLists.txt

```diff
@@ -581,6 +581,7 @@ foreach(module_name ${ZEPHYR_MODULE_NAMES})
   # https://cmake.org/pipermail/cmake/2019-June/069547.html
   zephyr_string(SANITIZE TOUPPER MODULE_NAME_UPPER ${module_name})
   if(NOT ${ZEPHYR_${MODULE_NAME_UPPER}_CMAKE_DIR} STREQUAL "")
+    set(ZEPHYR_CURRENT_MODULE_NAME ${module_name})
```

Comment:
Please clear `ZEPHYR_CURRENT_MODULE_NAME` like this:
https://github.com/zephyrproject-rtos/zephyr/blob/7fbcbcbfa7b0563aa16e48d3ae9c762ccd04debc/CMakeLists.txt#L591-L592